src/main.cpp

```diff
@@ -288,12 +288,13 @@ static char *tcmu_get_path(struct tcmu_device *dev) {
 static int dev_open(struct tcmu_device *dev) {
     char *config = tcmu_get_path(dev);
     LOG_INFO("dev open `", config);
-    if (!config)
-        return -EPERM;
+    if (!config) {
+        LOG_ERROR_RETURN(0, -EPERM, "get tcmu config failed");
```

Comment:
`tcmu config` is ambiguous, change to `get image config path failed` ?

Reply:
OK